include/fmt/format.h

```diff
@@ -1593,7 +1593,7 @@ template <typename OutputIt, typename Char, typename UInt> struct int_writer {
     format_decimal(digits, abs_value, num_digits);
     basic_memory_buffer<Char> buffer;
     size += prefix_size;
-    buffer.resize(size);
+    buffer.resize(to_unsigned(size));
```

Comment:
I suggest storing `to_unsigned(size)` in a variable (`usize`?) to avoid copy-pasta.

Reply:
Is adressed with e6159ed.